docs/tutorials/services/navigate-with-rover-base.md

```diff
@@ -0,0 +1,646 @@
+    },
+    {
+      "attributes": {
+        "board": "local",
+        "pins": {
+          "b": "lb",
+          "a": "la"
+        }
+      },
+      "depends_on": [],
+      "name": "l-encoder",
+      "type": "encoder",
+      "model": "incremental"
+    },
+    {
+      "type": "encoder",
+      "model": "incremental",
+      "attributes": {
+        "board": "local",
+        "pins": {
+          "b": "rb",
+          "a": "ra"
+        }
+      },
+      "depends_on": [],
+      "name": "r-encoder"
+    },
+    {
+      "model": "wheeled-odometry",
+      "type": "movement_sensor",
+      "namespace": "rdk",
+      "attributes": {
+        "base": "base",
+        "left_motors": [
+          "left-motor"
+        ],
+        "right_motors": [
+          "right-motor"
+        ]
+      },
+      "depends_on": [],
+      "name": "enc-linear"
+    }
+]
+```
+
+{{% /expand%}}
+
+## Configure a Navigation service
+
+Now, add the navigation service so that your wheeled base can navigate between waypoints and avoid obstacles.
+To add the navigation service to your robot, do the following:
+
+1. On your robot's **Config** page, navigate to the **Services** tab.
+2. At the bottom of the page, create a service.
+   Choose `Navigation` as the type.
+3. Then click **Create Service**.
+4. Select **Raw JSON** mode. Copy and paste the following into your new service's `"attributes"`:
+
+   ```json
+   {
+     "base": "base",
+     "movement_sensor": "merged",
+     "obstacles": [],
+     "store": {
+       "type": "memory"
+     },
+     "position_polling_frequency": 2,
+     "meters_per_sec": 1.2,
+     "degs_per_sec": 90,
+     "plan_deviation_m": 0.25
+   }
+   ```
+
+   Edit the attributes as applicable.
+   Attribute information is available in [the navigation service documentation](/services/navigation/#configuration).
+
+5. Click **Save Config** at the bottom of the window.
+
+Your navigation service should now appear in your robot's **Config** tab as a card with a map like the following:
+
+![Navigation Card](/tutorials/navigate-with-rover-base/navigation-config-builder.png)
+
+For more detailed information see [the navigation service](/services/navigation/#configuration).
+
+### Full JSON Configuration
+
+Now, at this point, if you switch to **Raw JSON** mode in your robot's **Config** tab, the full `"services"` array should look similar to the following:
+
+{{%expand "Click to view full example JSON" %}}
+
+```json {class="line-numbers linkable-line-numbers"}
+"services": [
+  {
+    "name": "nav",
+    "type": "navigation",
+    "attributes": {
+    "base": "base",
+    "movement_sensor": "merged",
+    "obstacles": [],
+    "store": {
+        "type": "memory"
+    },
+    "position_polling_frequency": 2,
+    "meters_per_sec": 1.2,
+    "degs_per_sec": 90,
+    "plan_deviation_m": 0.25
+    }
+  }
+]
+```
+
+{{% /expand%}}
+
+## Start navigating with the Navigation service
+
+Next, add waypoints to your navigation service.
```

Comment:
Instead of starting sentences with "Now" or "Next" that do not convey any useful information as a transition - we can add transitions that help readers put together how things connect.
```suggestion
Now that you have configured your navigation service, add waypoints to your navigation service.
You can add waypoints from the [Control tab](tutorials/services/navigate-with-rover-base/#control-tab-method) or [programmatically](/tutorials/services/navigate-with-rover-base/#programmatic-method).
```

Reply:
Done